sktime/transformations/series/holiday.py

```diff
@@ -0,0 +1,310 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements holidays transformers."""
+import pandas
+
+from sktime.transformations.base import BaseTransformer
+
+__all__ = ["CountryHolidaysTransformer", "FinancialHolidaysTransformer"]
+__author__ = ["yarnabrina"]
+
+
+class CountryHolidaysTransformer(BaseTransformer):
+    """Country Holidays Transformer.
+
+    This implementation wraps over holidays [1]_ by dr-prodigy.
+
+    Parameters
+    ----------
+    country : str
+        An ISO 3166-1 Alpha-2 country code.
+    subdiv : str, optional
+        The subdivision (e.g. state or province); not implemented for all countries
+        (see documentation).
```

Comment:
which documentation? Link would be useful.

Perhaps in the preamble, link to the original package documentation for country code register and subivisions?